### PR TITLE
Update issue and PR templates

### PR DESCRIPTION
**Pull request by carmocca**
## What does this PR do?

A set of random fixes and improvements to our templates.

The feature-request projects text is now smaller.

### Does your PR introduce any breaking changes ? If yes, please list them.

None

## Before submitting
- [ ] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [ ] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [x] Did you make sure to **update the documentation** with your changes? (if necessary)
- [n/a] Did you write any **new necessary tests**? (not for typos and docs)
- [x] Did you verify new and **existing tests pass** locally with your changes?
- [n/a] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or internal minor changes/refactorings)
- [x] Did you list all the **breaking changes** introduced by this pull request?

## PR review

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -10,37 +10,42 @@ assignees: ''
...
-
-* `Colab Notebook`: Please copy and paste the output from our [environment collection script](https://raw.githubusercontent.com/PyTorchLightning/pytorch-lightning/master/requirements/collect_env_details.py) (or fill out the checklist below manually).
+<!--
+Please copy and paste the output from our environment collection script:
+https://raw.githubusercontent.com/PyTorchLightning/pytorch-lightning/master/requirements/collect_env_details.py
+(For security purposes, please check the contents of the script before running it)
 
 You can get the script and run it with:
-```
+```console
```
Comment (Borda) [inline .github/ISSUE_TEMPLATE/bug_report.md:41]:
```suggestion
```bash
```
but probably it is te same

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -26,10 +27,18 @@ assignees: ''
...
+---
+
+#### If you enjoy Lightning, check out our other projects! ⚡
+
+<sub>
+
+- [**Metrics**](https://github.com/PyTorchLightning/metrics): Machine learning metrics for distributed, scalable PyTorch applications.
+
+- [**Flash**](https://github.com/PyTorchLightning/lightning-flash): The fastest way to get a Lightning baseline! A collection of tasks for fast prototyping, baselining, finetuning and solving problems with deep learning
+
```
Comment (Borda) [inline .github/ISSUE_TEMPLATE/feature_request.md:39]:
```suggestion
- [**Flash**](https://github.com/PyTorchLightning/lightning-flash): The fastest way to get a Lightning baseline! A collection of tasks for fast prototyping, baselining, finetuning and solving problems with deep learning
```
list is without spaces, right?

**Review by Borda — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (carmocca) [reply .github/ISSUE_TEMPLATE/feature_request.md:39]: The `sub` block doesnt render properly without the spaces

**Review by tchaton — APPROVED** [review]
LGTM !

**Review by awaelchli — APPROVED** [review]
